src/main/java/com/sda/microblogging/controller/PostController.java

```diff
@@ -0,0 +1,78 @@
+package com.sda.microblogging.controller;
+
+import com.sda.microblogging.entity.*;
+import com.sda.microblogging.entity.DTO.post.PostSaveDTO;
+import com.sda.microblogging.entity.DTO.post.PostDTO;
+import com.sda.microblogging.entity.mapper.PostMapper;
+import com.sda.microblogging.service.PostLikeService;
+import com.sda.microblogging.service.PostService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/home")
```

Comment:
The base path for post should be post

Reply:
PostController shall start with
@RequestMapping(path = "/microblogging/v1/post")
Now it even does not appear in swagger..